src/platyPS/platyPS.psm1

```diff
@@ -2603,6 +2636,9 @@ function ConvertPsObjectsToMamlModel
                         # https://github.com/PowerShell/platyPS/issues/211
                         'Confirm' { "Prompts you for confirmation before running the cmdlet.`r`n`r`n" }
                         'WhatIf' { "Shows what would happen if the cmdlet runs. The cmdlet is not run.`r`n`r`n" }
+                        'IncludeTotalCount' { "Reports the number of objects in the data set (an integer) followed by the objects. If the cmdlet cannot determine the total count, it returns 'Unknown total count'.`r`n`r`n" }
```

Comment:
I don't think that `IncludeTotalCount` is very common - I didn't see it used much. Can you point to some examples outside of Armor module?

Reply:
Sure thing.  I haven't implemented this in the Armor module quite yet, but have in my new PSRyver module and here's [339 other examples](https://github.com/search?l=PowerShell&q=supportspaging+%3D+%24true&type=Code) of the [SupportsPaging](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_cmdletbindingattribute?view=powershell-6#supportspaging) PowerShell v3.0 CmdletBinding attribute that I found via a quick search in GitHub.  Definitely not as common as `SupportsShouldProcess`, but they are built-in parameters like `WhatIf` and `Confirm`.

Also, here's what the `Get-Help -Full` output looks like:
```powershell
    -IncludeTotalCount [<SwitchParameter>]

        Required?                    false
        Position?                    named
        Default value
        Accept pipeline input?       false
        Accept wildcard characters?  false

    -Skip <UInt64>

        Required?                    false
        Position?                    named
        Default value
        Accept pipeline input?       false
        Accept wildcard characters?  false

    -First <UInt64>

        Required?                    false
        Position?                    named
        Default value
        Accept pipeline input?       false
        Accept wildcard characters?  false
```

Reply:
Ah I see, so the `-IncludeTotalCount` is part of the `SupportPagging`. Interesting! Ok that makes sense now 👍